andela/ah-the-unsullied-frontend
# #162949174 Implement social login on the frontend

#### What does this PR do?
Implements social login on the front end with Facebook, Google and Twitter
#### Description of Task to be completed?
- Users should be able to log in to the system using their social accounts with Facebook, Google and Twitter
#### How should this be manually tested?
1. Clone the repo: `git clone ah-the-unsullied-frontend.git`
2. Checkout to the social login branch: `git checkout ft-social-login-162949174`
3. Install the requirements: `npm install` or `yarn`
4. Create a .env file with the following environment variables: 
```
REACT_APP_API_KEY='AIzaSyBfD1DxcoCvCotR7q3BmyUWJzVycOb8OLA'
REACT_APP_AUTH_DOMAIN='anotherproject-77039.firebaseapp.com'
```
5. Export the variables on the terminal: `source .env`
6. Run the tests:  `yarn test`
7. Run the app: `yarn start`
The app should start. You should be able to authenticate using your Facebook, Google or Twitter account
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#162949174](https://www.pivotaltracker.com/story/show/162949174)
